src/main/resources/reference.conf

```diff
@@ -63,6 +69,9 @@ resourceTypes = {
         description = "perform writer actions on the workspace"
         authDomainConstrainable = true
       }
+      "edit" = {
```

Comment:
I don't think so, I don't know when we would check an application action.
Rather than asking Sam "is this user an application", I think we'll probably ask e.g. "can this user create an application-owned shared resource"